p2p/p2ptest/network.go

```diff
@@ -197,8 +197,8 @@ type Node struct {
 	Transport   *p2p.MemoryTransport
 }
 
-// MakeNode creates a new Node.
-func MakeNode(t *testing.T, network *Network) *Node {
+// AddNode creates a new Node and adds it to the network.
```

Comment:
but is it adding it to the network?

Reply:
yeah, good catch, I've updated the docstring and renamed the function.